src/cpu/aarch64/matmul/acl_matmul_utils.cpp

```diff
@@ -84,11 +84,8 @@ status_t init_conf_matmul(acl_matmul_conf_t &amp, memory_desc_t &src_md,
     } else {
         auto src_tag = memory_desc_matches_one_of_tag(
                 src_md, acdb, abcd, abdc, abc, acb, ab, ba);
-        auto wei_tag = memory_desc_matches_one_of_tag(
```

Comment:
how is this different from the previous if block (Lines 77 to 83) which deals with the FixedFormat case? I guess the check for plain formats is removed to support passing blocked format Ab8a via wtag?

What happens when you choose a different blocked format other than --wtag=Ab8a ? e.g --wtag=aBdc8b  or ABcd8b8a ?

Reply:
Hi @cfRod , this is slightly different from the regular fixed format scenario. This is required to support precompiled graphs where the primitive gets created with the reordered (already reordered) weight tensors, so their formats are blocked and more custom. 

btw, it's not just the `Ab8a`... format, even the `ab`, ` acbd` etc are coming as blocked formats, thats why I combined the logic with the `else` part (Lined 84 to 87) instead of the format::any.

I had added a similar support to `inner_product` primitive sometime back, here is the PR, please check for more context. it was more straight forward there because everything was handled as a fixed format there
https://github.com/oneapi-src/oneDNN/pull/1768